core/constraints/constraints.go

```diff
@@ -23,20 +23,21 @@ const (
 	Arch      = "arch"
 	Container = "container"
 	// cpuCores is an alias for Cores.
-	cpuCores         = "cpu-cores"
-	Cores            = "cores"
-	CpuPower         = "cpu-power"
-	Mem              = "mem"
-	RootDisk         = "root-disk"
-	RootDiskSource   = "root-disk-source"
-	Tags             = "tags"
-	InstanceRole     = "instance-role"
-	InstanceType     = "instance-type"
-	Spaces           = "spaces"
-	VirtType         = "virt-type"
-	Zones            = "zones"
-	AllocatePublicIP = "allocate-public-ip"
-	ImageID          = "image-id"
+	cpuCores             = "cpu-cores"
+	Cores                = "cores"
+	CpuPower             = "cpu-power"
+	NestedVirtualization = "nested-virtualization"
```

Comment:
You'd need to update all of the other providers around their `unsupportedConstraints =` values

Reply:
Or should it be part of `virt-type` instead?

Reply:
To answer my own question, no, it's not `virt-type`, the idea is to day  "this machine supports nested-kvm" per @addyess